website/docs/r/csi_volume.html.markdown

```diff
@@ -137,5 +137,22 @@ configuration options.
 - `create` `(string: "10m")` - Timeout when creating or updating a new CSI volume.
 - `delete` `(string: "10m")` - Timeout when deleting a CSI volume.
 
+## Importing CSI Volumes
+
+CSI volumes are imported using the pattern `<volume ID>@<namespace>` .
```

Comment:
I think these might benefit from some info on how to get the namespace, or short of that, saying ~"If you're not sure, it's probably the default, which is `@default`" and maybe using `@default` in the examples.

Reply:
Hum...not sure about this. If you're importing a resource you must know its namespace. If you don't, you really shouldn't be importing it 😬 

You will also need to set the namespace in the TF config, so you _really_ need to know it if you're trying to import it.

Reply:
For jobs, the namespace can be excluded from the jobspec, and it isn't an argument in the TF resource.  For CSI volumes, the namespace is a TF resource argument, but can also be left out and defaults to "default"

So our just-getting-started type folks, or even experienced users who just never needed namespaces for some reason, could totally get away with never thinking about namespaces at all -- the default "just works" -- so I'm thinking this might be a little learning opportunity where they could use just a bit of extra help.

Reply:
But importing a resource in Terraform is not targeted at just-getting-started folks, it's more like a day 10321 type of thing 😅